runtime/integration-tests/src/generic/cases/liquidity_pools.rs

```diff
@@ -6231,7 +6187,7 @@ mod altair {
 			env.parachain_state_mut(|| {
 				assert_eq!(
 					<CurrencyIdConvert as C1<_, _>>::convert(tranche_multilocation),
-					Err(tranche_multilocation),
+					None,
```

Comment:
Wondering whether we truly can ignore error propagation when deriving the CurrencyId from a location

Reply:
The why is similar to the above answer:

> Nothing forces you now to use TryConvert, so now the implementation can be a simple standalone function. Now it returns an Option instead of a Result. But note that the previous error used in the result was the same input given. So, actually, there was no extra information on it.

Also the traits where this type is required expect an `Option` now